kotti/util.py

```diff
@@ -175,12 +175,13 @@ def get_visible_children(self, context, request):
 class Link(LinkBase):
     template = 'kotti:templates/edit/el-link.pt'
 
-    def __init__(self, name, title=None, predicate=None):
+    def __init__(self, name, title=None, predicate=None, target=None):
```

Comment:
My first not commited version was implemented with ``target='_self'`` but I've seen that when chameleon templates attributes are set to None they are not generated at all. So if you don't provide any ``target``the ``_self``is implicit (correct me if I'm wrong!). This way we avoid an unneeded target="_self" because it is already the default. That's why I reverted my first implementation. Does it sound good for you?